scribble-doc/scribblings/scribble/demo-manual.scrbl

```diff
@@ -76,7 +72,9 @@
 
  @item{@css{RktModLink} (a linked module reference): @racketmodname[racket/base]}
 
- @item{@css{RktOpt} (option-argument brackets): brackets in @opt-example[]}
+ @item{@css{RktOpt} (option-argument brackets): brackets in
+
+       @defproc[(f [x any/c 1]) any]}
```

Comment:
This will create a "no declared exporting libraries for definition". I think there must be some way to render a `defproc` without link trouble, but I'm not sure of the right combination. Probably `#:link-target? #f` is part of the answer.